python/composio/tools/toolset.py

```diff
@@ -41,7 +41,7 @@
     TriggerModel,
     TriggerSubscription,
 )
-from composio.client.enums import TriggerType
+from composio.client.enums import TriggerType, app
```

Comment:
There's an unused import `app` that should be removed. The `App` class is already imported in the new import line.